README.md

```diff
@@ -26,3 +26,12 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+$$T(n)=\left{ 1 if n \leq 1, 3T(n/3) if n > 1\right}$$
```

Comment:
Hmm, looks like something is missing here. There's some work needed to combine the results from the recursive call that's missing here.

Reply:
So I added in the "+1" for that operation, is that what I'm missing? Does the addition of the recursive calls count as one or would you count it as 2 since there are 2 operations performed?

Reply:
Asymptotically it doesn't really matter whether you count it as 1 or 2.